distribution/docs/src/main/resources/content/_installing/java-reqs.adoc

```diff
@@ -22,8 +22,13 @@ For a development system:
 . Set the `JAVA_HOME` environment variable to the location where the JDK is installed.
 
 
+[NOTE]
+====
+Prior to installing ${branding}, ensure the system time is accurate to prevent federation issues.
+====
+
 .*NIX Unlinking JAVA_HOME if Previously Set
-[WARNING]
+[NOTE]
 ====
 Unlink `JAVA_HOME` if it is already linked to a previous version of the JRE:
```

Comment:
I know this didn't change, but I'm not sure what it means to unlink `JAVA_HOME` home. When I read unlink,I think "use `unlink` to remove a symbolic link. Is that what this means? If so, which link? Or does it mean to use `unset` to unset the shell variable named `JAVA_HOME`? I think the documentation should be clear on this.

Reply:
right below this it says:

`unlink JAVA_HOME`

Reply:
I can't think why someone would create a symbolic link named JAVA_HOME in the first place. JAVA_HOME should be the name of shell var.

Reply:
changing to unset